src/Tester.js

```diff
@@ -63,7 +72,7 @@ export default class Tester extends Component {
   }
 
   async runTests() {
-    scope = new TestScope(this, this.props.waitTime);
+    let scope = new TestScope(this, this.props.waitTime, this.props.testStartDelay, this.props.consoleLog, this.props.reporter, this.props.notifier, this.props.reRender);
```

Comment:
Adding a whole bunch of sliced out props as arguments here feels a bit nasty. Would an object be a better API?